### PR TITLE
Cloud Manager 2022.11.0 API Docs

### PR DESCRIPTION
**Pull request by negruta**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: swagger-specs/api.yaml [hunk 1]
```diff
@@ -5503,7 +5508,7 @@ definitions:
       region:
         type: string
         example: va6
-        description: Region to create the infrastructure
+        description: 'Region to create the infrastructure. Use your primary region, whose identifier can be retrieved from the List Regions endpoint.'
```
Comment (cosminseceleanu) [inline swagger-specs/api.yaml:5511]: not sure whether `List Regions endpoint` is available for public api

Comment (negruta) [reply swagger-specs/api.yaml:5511]: The API is public: https://developer.adobe.com/experience-cloud/cloud-manager/reference/api/#tag/Regions

**Review by panchapa — APPROVED** [review]

File: swagger-specs/api.yaml (hunk 1 above)
Comment (bohnertchris) [reply swagger-specs/api.yaml:5511]: Since you cite the Use Regions API as useful in this case, I would suggest adding a link to it for the user.

Comment (negruta) [reply swagger-specs/api.yaml]: @bohnertchris I like your suggestion. I updated the PR this change. Please review

Comment (bohnertchris) [reply swagger-specs/api.yaml]: LGTM

**Review by ctinmota — APPROVED** [review]

**Review by bohnertchris — APPROVED** [review]
LGTM

**Comment by bohnertchris** [comment]
@cosminseceleanu, would you mind completing your review? Thanks.

**Review by cosminseceleanu — APPROVED** [review]
LGTM

**Pull request merged**